os/windows/sign.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+REM Signing script
+REM Arguments: sign.bat exe_to_sign certificate_subject_name
+
+IF NOT DEFINED SIGNTOOL_PATH (SET SIGNTOOL_PATH=signtool)
+IF NOT DEFINED SIGNTOOL_TIMESTAMP_URL (SET SIGNTOOL_TIMESTAMP_URL=http://timestamp.digicert.com)
```

Comment:
`http`? `digicert`? No clue what this is, but it reads weird :)

Reply:
Lol, okay, so this is in the official Microsoft documentation:
https://docs.microsoft.com/en-us/dotnet/framework/tools/signtool-exe

It feels really odd, to have a `http` mixed in with signing .. but what-ever :P I would suggest we just add a comment above, saying the URL comes from https://docs.microsoft.com/en-us/dotnet/framework/tools/signtool-exe and leave it at that :)

Reply:
It's used to timestamp the file with a cryptographic signature; there are a variety of servers available. Agreed that it does seem a bit odd that it's not HTTPS. I'll comment it a bit more.